### PR TITLE
Remove redundant PreArm: text

### PR DESCRIPTION
**Pull request by peterbarker**
This is on top of https://github.com/ArduPilot/ardupilot/pull/10711 and https://github.com/ArduPilot/ardupilot/pull/10710

Still not sure how to fix the AP_Motors ones.

### CODE REVIEW

**Comment by rmackay9** [comment]
Looks like it needs a rebase..

File: APMrover2/AP_MotorsUGV.cpp [hunk 1]
```diff
@@ -401,33 +401,25 @@ bool AP_MotorsUGV::pre_arm_check(bool report) const
         SRV_Channels::function_assigned(SRV_Channel::k_throttleRight) &&
         SRV_Channels::function_assigned(SRV_Channel::k_throttle) &&
         SRV_Channels::function_assigned(SRV_Channel::k_steering)) {
-        if (report) {
-            gcs().send_text(MAV_SEVERITY_CRITICAL, "PreArm: regular AND skid steering configured");
-        }
+        rover.arming.check_failed(ARMING_CHECK_NONE, report, "regular AND skid steering configured");
```
Comment (rmackay9) [inline APMrover2/AP_MotorsUGV.cpp]: we should probably be passing back a failure string.. it's OK for now but..

Comment (peterbarker) [reply APMrover2/AP_MotorsUGV.cpp]: Should we have entry points into the motors library for each different bit which might be checked 'though?

`So motors.prearm_checks_parameters(...)` `motors.prearm_checks_not_really_covered_by_other_bits` etc?

MdB has already said the same thing as you, so I might just drop the patches for now.

Comment (rmackay9) [reply APMrover2/AP_MotorsUGV.cpp]: sure, leaving it out seems fine to me.

File: APMrover2/AP_Arming.cpp [hunk 2]
```diff
@@ -49,9 +49,7 @@ bool AP_Arming_Rover::gps_checks(bool display_failure)
 
     // check for ekf failsafe
     if (rover.failsafe.ekf) {
-        if (display_failure) {
-            gcs().send_text(MAV_SEVERITY_CRITICAL, "PreArm: EKF failsafe");
-        }
+        check_failed(ARMING_CHECK_NONE, display_failure, "EKF failsafe")
```
Comment (rmackay9) [inline APMrover2/AP_Arming.cpp]: travis says a ";" is missing here..

Comment (peterbarker) [reply APMrover2/AP_Arming.cpp]: Fixed.

File: APMrover2/AP_MotorsUGV.cpp (hunk 1 above)
Comment (peterbarker) [reply APMrover2/AP_MotorsUGV.cpp]: Dropped those.  We should come back to them.

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**